tests/OperationStore.spec.ts

```diff
@@ -64,21 +71,69 @@ async function constructAnchoredUpdateOperation (
     }]
   };
 
-  const updateOperationBuffer = await OperationGenerator.generateUpdateOperation(updatePayload, '#key1', privateKey);
+  const updateOperationBuffer = await OperationGenerator.generateUpdateOperationBuffer(updatePayload, '#key1', privateKey);
   return constructAnchoredOperation(updateOperationBuffer, transactionNumber, transactionTime, operationIndex);
 }
 
+const databaseName = 'sidetree';
+const operationCollectionName = 'operations';
+
+/**
+ * Test if a mongo service is running at the specified url
+ */
+async function isMongoServiceAvailable (serverUrl: string): Promise<boolean> {
+  try {
+    const client = await MongoClient.connect(serverUrl);
+    await client.close();
+  } catch (error) {
+    console.log('Mongoclient connect error: ' + error);
+    return false;
+  }
+  return true;
+}
+
+/**
+ * Clear a mongo collection - used to remove test state.
+ */
+async function clearMongoCollection () {
+  const client = await MongoClient.connect('mongodb://localhost:27017');
+  const db = client.db(databaseName);
+  const collections = await db.collections();
+  const collectionNames = collections.map(collection => collection.collectionName);
+
+  // If the operation collection exists, use it; else create it then use it.
```

Comment:
Sanity: I don't see the else portion? Also are you able to use the collection if it is dropped?

Reply:
Sorry, cut-n-paste legacy. the comment is invalid. Fixed it and renamed the function.